LICENSE

```diff
@@ -0,0 +1 @@
+Permission is granted to use/copy/modify the contents of this repository without restriction.
```

Comment:
why not just use one of the standard licenses, eg see https://github.com/dlang-community/D-Scanner/blob/master/LICENSE_1_0.txt

Reply:
I don't know of a standard license that is as free as what I've written here...do you?

Reply:
Well, there is WTFPL, but I don't want to put people off by including a "curse word" in my license.